src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -937,7 +943,7 @@ def assign_college_simulants_guardians(
             .reset_index()
             .rename(columns={"index": "partner_id"})
             .groupby("household_id")["partner_id"]
-            .apply(np.random.choice)
+            .first()
```

Comment:
Note: There should never be more than one item to choose from here. See #225 for a new integration test of this household structure invariant.

The `np.random.choice` version was failing for me with Pandas 1.4.0 due to this bug: pandas-dev/pandas#47985

Reply:
Why was random.choice ever the thing if there was always only one option? Or did that used to not be the case?

Reply:
My best guess is that it wasn't known by the person implementing this whether ACS source data would always respect this constraint -- it does.